starknet_py/net/client_errors.py

```diff
@@ -8,9 +8,15 @@ class ClientError(Exception):
 
     def __init__(self, message: str, code: Optional[str] = None):
         self.code = code
-        self.message = f"Client failed{f' with code {code}' if code is not None else ''}: {message}"
+        self.message = message.strip()
```

Comment:
Is the strip() necessary? What if for some reason the user is expecting trailing whitespace

Reply:
It seems to be always JSON encoded with a trailing `\n` when it comes from the server.
What would be the use case to add a trailing whitespace?

Reply:
Maybe something like this, once strings are more of a thing in Cairo:

```cairo
"Expected number but receiver value: "
```

in the case when a user might supply whitespace as an argument and the whitespace gets interpolated in the error message

Reply:
In that case, the full error would be something like:
```json
{
    "code": 42,
    "message": "Expected number but received value: "
}
```
And so the `strip() applies only to the stringified response:

```json
{"code":42,"message":"Expected number but received value: "}\n
```

I may miss some specific usages though. So if it's problematic, let's remove that `strip()` :)

Reply:
Oh wait, you're right. Sorry :sweat: 
I'll remove it.